features/cmake/find_package/find_cmake_multi_generator_targets/bye/src/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(PackageTest CXX)
+
```

Comment:
How this CMakeLists.txt finds the generated find.cmake file? There is nothing here like in the consumer PROJECT ``set(CMAKE_PREFIX_PATH ${CMAKE_BINARY_DIR})``. Why necessary in the project but not here?

Reply:
The recipe for "bye" is declaring ` generators = "cmake_find_package_multi"` so the `CMake()` build helper knows it and automatically adjust the `CMAKE_PREFIX_PATH` automatically. Actually, there is a but there, it should adjust the `CMAKE_MODULE_PATH` too. I'll open it. But for this case is enough